roles/common/tasks/monitoring.yml

```diff
@@ -61,7 +61,7 @@
 
 - name: ensure PATH contains /usr/local/bin (OpenStack clients)
   lineinfile: dest=/etc/default/sensu regexp=^PATH
```

Comment:
systemd/centos services usually put defaults in `/etc/sysconfig/...` instead of `/etc/default`  we should confirm that `/etc/default/sensu` is correct for centos here.